script/sugar-install.sh

```diff
@@ -64,11 +64,7 @@ if [ "$OS_FLAVOUR" = Darwin ]; then
     esac
 fi
 
-DIST="$VERSION.zip"
-
-if [ "$VERSION" = "macos-intel-latest" ]; then
-    DIST="macos-latest.zip"
-fi
+DIST="$VERSION"
```

Comment:
The zip fetching variable is changed to just the binary name and the check to change `macos-intel-latest` as `macos-latest` is removed as the binary names aren't changed underneath